### DISCUSSION TITLE
Release Candidate 1

### DISCUSSION THREAD

**Discussion by DannybotDev** [FunnyDannyG/FNFMegaMayhem#17] [Announcements]
Please look over this release extensively and provide bug reports to Danny

<hr /><em>This discussion was created from the release <a href='https://github.com/DannybotDev/FNFMegaMayhem/releases/tag/v2-RC1'>Release Candidate 1</a>.</em>

**Comment by JordiWifBread** [comment]
Hello gamers

**Comment by JordiWifBread** [comment]
What did you have for breakfast today?
